app/parse.py

```diff
@@ -8,9 +14,131 @@
     tags: list[str]
 
 
-def main(output_csv_path: str) -> None:
-    pass
+@dataclass
+class Author:
+    name: str
+    birth_date: str
+    birth_location: str
+    decription: str
+
+
+BASE_URL = "https://quotes.toscrape.com"
+
+QUOTE_FIELDS = [field.name for field in fields(Quote)]
+AUTHOR_FIELDS = [field.name for field in fields(Author)]
+
+
+cached_author_links = set()
+
+
+def parse_single_quote(quote_soup: Tag) -> Quote:
+    text = quote_soup.select_one(".text").text
+    author = quote_soup.select_one(".author").text
+    tags = [tag.text for tag in quote_soup.select(".tag")]
+
+    quote = Quote(
+        text=text,
+        author=author,
+        tags=tags,
+    )
+
+    author_link = quote_soup.select_one("span a").get("href")
+    cached_author_links.add(f"{author_link}/")
```

Comment:
The code adds a trailing slash to `author_link` before adding it to `cached_author_links`. Ensure that this matches the format expected later in the code when constructing URLs.